src/Google/RequestReviewStatuses.php

```diff
@@ -24,15 +24,21 @@ class RequestReviewStatuses implements Service {
 	 * @return array
 	 */
 	public function get_statuses_from_response( array $response ) {
-		$issues         = [];
-		$review_allowed = false;
-		$status         = 'APPROVED';
+		$issues   = [];
+		$cooldown = '';
+		$status   = 'APPROVED';
 
 		foreach ( $response as $program_type ) {
 			foreach ( $program_type['data']['regionStatuses'] as $region_status ) {
 
-				if ( $region_status['reviewEligibilityStatus'] === self::ELIGIBLE ) {
-					$review_allowed = true;
+				if (
+					isset( $region_status['reviewIneligibilityReasonDetails'] ) &&
+					isset( $region_status['reviewIneligibilityReasonDetails']['cooldownTime'] )
+				) {
+					$region_cooldown = $region_status['reviewIneligibilityReasonDetails']['cooldownTime'];
+					if ( ! $cooldown || $region_cooldown > $cooldown ) {
```

Comment:
> Also, I guess that once the cooldownTime period is reached, this property is not returned anymore, is this right? 

Correct, one the cool down period is ended, then the status will be ELIGIBLE and that property is not available anymore thats why I don't check with todays time.

Reply:
If it is in epoch format, I do not think this should be treated as a string,  as the epoch represents the number of seconds that have elapsed since the Unix epoch. 

What do you think?

Reply:
Thanks, @puntope for the changes, yeah it will be more clear when we know the exact response from Google.